src/apps/AppCenter/InstalledApps/AppContent.js

```diff
@@ -27,15 +29,30 @@ const AppContent = React.memo(({ repo, repoVersions, onRequestUpgrade }) => {
         description = '',
         screenshots = [],
         source_url: sourceUrl,
+        details_url: detailsUrl,
       },
       version: latestVersion,
     },
   } = repo
+  const [repoDetails, setRepoDetails] = React.useState(null)
+  React.useEffect(() => {
+    const fetchDescription = async () => {
+      try {
+        const raw = await fetch(`${baseUrl}${removeStartingSlash(detailsUrl)}`)
+        const res = await raw.text()
+        setRepoDetails(res)
+      } catch (e) {
+        console.log('Error fetching decription: ', e)
+      }
+    }
+    fetchDescription()
```

Comment:
Should we also add a cancel flag (@bpierre did it a few other places using `useEffect()`) so that an old fetch doesn't accidentally replace a new one?

Reply:
Let me see how I can update this; for this to happen a first fetch has to take some time, then the `details_url` has to change to trigger the second fetch and this 2nd one would need to resolve before the 1st one..

Reply:
Added `usePromise` hook to make this request _cancellable_: https://github.com/aragon/aragon/pull/750/commits/d39085f3f4c616cbe8732e5c0496811a04b98328 and https://github.com/aragon/aragon/pull/750/commits/9bec3aab4f33598c65aaba3f8429b64d16af1ed4